### ISSUE TITLE
Network errors with extension version 1.6.9 in Waterfox

### ISSUE THREAD

**Issue by grahamperrin** [Rob--W/crxviewer#75]
### Example

Using the context menu at <https://addons.mozilla.org/en-GB/firefox/addon/monochromer/> leads to `moz-extension://3ab1bc04-233d-498c-9bd2-4112b193b66e/crxviewer.html?crx=https%3A//addons.mozilla.org/firefox/downloads/latest/monochromer/platform%3A2/monochromer.xpi` and: 

> Network error for https://addons.mozilla.org/firefox/downloads/latest/monochromer/platform:2/monochromer.xpi

### Reproducibility

Not reproducible with 1.6.9 in Firefox on the same notebook. 

https://robwu.nl/crxviewer/ in Waterfox is fine. When fed with https://addons.mozilla.org/en-GB/firefox/addon/monochromer/ the result is https://robwu.nl/crxviewer/?crx=https%3A%2F%2Faddons.mozilla.org%2Fen-GB%2Ffirefox%2Faddon%2Fmonochromer%2F

### Environment

[Waterfox](http://www.waterfoxproject.org/) 56.2.5.

FreeBSD-CURRENT. 

`Mozilla/5.0 (X11; FreeBSD amd64; rv:56.0; Waterfox) Gecko/20100101 Firefox/56.2.5`

Side note: on this unsupported platform, I'm (contentedly) without the AMO-specific overrides that are under https://github.com/MrAlex94/Waterfox/commit/8e978a6df522a2e12ab3080d691fec490db01a56#diff-61cac4855c1bb1dd3e264ed799753d4fR30

```text
$ date ; uname -v
Sat 24 Nov 2018 08:55:03 GMT
FreeBSD 13.0-CURRENT r340689 GENERIC 
$ poudriere jail -i -j current | grep -i version
Jail version:      13.0-CURRENT 1300003
Jail vcs version:  r340689
$ pkg info waterfox | grep -i repository
        repository     : poudriere
$ pkg info firefox | grep -i repository
        repository     : poudriere
$ pkg query '%o %v %R' waterfox firefox
www/waterfox 56.2.5 poudriere
www/firefox 63.0.3_3,1 poudriere
$ 
```

### Noted

['Network error' since bug 1415644 · Issue #58 …](https://github.com/Rob--W/crxviewer/issues/58) (closed) with reference to Mozilla bug [1450649 - Extension cannot fetch data from addons.mozilla.org because the host permissions are not applied](https://bugzilla.mozilla.org/show_bug.cgi?id=1450649) (RESOLVED WONTFIX).

**Comment by Rob--W** [comment]
Can you show the output of the developer console (press F12 while being in the moz-extension: page in Firefox/Waterfox, then reload the page)? Even a screenshot is probably fine, as I am interested in seeing whether a CORS message appears.

**Comment by curiosity-seeker** [comment]
I had a Network Error also with Firefox 63. After inspecting the developer console I found out that d1sp2sgy246t7c.cloudfront.net was blocked by one of the filterlists in my Pi-hole. After whitelisting that domain all is well. It's possible that this also causes @grahamperrin 's problem.

**Comment by Rob--W** [comment]
Published update. I assume that this issue has been fixed; if not, please let me know.
